setup.py

```diff
@@ -245,7 +245,7 @@ def write_version(filename=os.path.join(*['airflow',
     'lxml>=4.0.0',
     'mock',
     'mongomock',
-    'moto==1.1.19',
+    'moto==1.3.6',
```

Comment:
Can we keep this at 1.3.5 for now? There was a lot of changes around how moto worked around 1.3.5-1.3.6/7 and I'l like to stay on the same side of that change for now.